src/2982R0_std_quantity_as_a_numeric_type.md

```diff
@@ -2284,10 +2287,90 @@ if (is_neq_zero(q1 / q2))
   // ...
 ```
 
-Those functions will work with any type `T` that exposes `zero()` member function returning
+Those functions will work with any type `T` that exposes a `zero()` member function returning
 something comparable to `T`. Thanks to that, we can use them not only with quantities but also
 with quantity points, `std::chrono::duration` or any other type that exposes such an interface.
 
+#### Alternative: `Zero` type
+
+The [@AU] library takes a different approach to this problem.  It provides an empty type, `Zero`, which
+represents a value of exactly 0 (in any units).  It also provides an instance `ZERO` of this type.  Every
+quantity is implicitly constructible from `Zero`.
+
+Consider this example legacy (i.e., pre-units-library) code:
+
+```cpp
+if (speed_squared_m2ps2 > 0) { /* ... */ }
+```
+
+When users upgrade to a units library, they will replace the raw number `speed_squared_m2ps2` with a strongly
+typed quantity `speed_squared`.  Unfortunately, this replacement won't compile, because quantities can't be
+constructed from raw numeric values such as `0`.  They can fix this problem by using the instance, `ZERO`,
+which encodes its value _in the type_:
+
+```cpp
+if (speed_squared > ZERO) { /* ... */ }
+```
+
+This has significant advantages.  It preserves the _form_ of the code, making the transition less error prone
+than replacement with a function such as `is_gt_zero`.  It also reduces the number of new comparison APIs
+a user must learn: `Zero` handles them all.
+
+`Zero` has one downside: it will not work when passed across _generic quantity_ interfaces.  `Zero`'s value
```

Comment:
```suggestion
`Zero` has one downside: it will not work when passed across _generic quantity_ interfaces.  The benefit of `Zero`
```
To not confuse it with the value `ZERO`, value of `Zero`.